src/main/java/com/javachallengers/simpson/controller/RestControllerExceptionHandler.java

```diff
@@ -0,0 +1,35 @@
+package com.javachallengers.simpson.controller;
+
+import java.util.Date;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.server.ResponseStatusException;
+
+import com.javachallengers.simpson.model.dto.ApiErrorDTO;
+
+@ControllerAdvice
+public class RestControllerExceptionHandler {
+	private static final Logger logger = LoggerFactory.getLogger(RestControllerExceptionHandler.class);
+	
+	@ExceptionHandler(value = { ResponseStatusException.class })
+	public ResponseEntity<ApiErrorDTO> handleResponseStatusException(ResponseStatusException ex) {
+		logger.error(ex.getMessage(), ex);
+		
+		String error = ex.getStatus().getReasonPhrase();
+		int status = ex.getStatus().value();
+		String message = ex.getReason();
+		long timestamp = new Date().getTime();
```

Comment:
Is possible use UTC instead of machine date time? UTC is useful for save/record and when the client get the data is able to transform to your correct timezone.